Docs/developer_guide/build_instructions/linux.md

```diff
@@ -76,6 +76,41 @@ sudo apt update && sudo apt install git subversion build-essential cmake cmake-c
   qtbase5-private-dev libqt5x11extras5-dev libxt-dev
 ```
 
+### Ubuntu 23.04 (Lunar Lobster)
+
+:::{warning}
+If you get cmake version error, download build and install cmake 3.25.3:
+```console
+# Remove previous cmake versions
+sudo apt remove cmake
+sudo apt purge --auto-remove cmake
+
+sudo apt-get install build-essential
+wget https://cmake.org/files/v3.25/cmake-3.25.3.tar.gz
+tar xf cmake-3.25.3.tar.gz
+
+cd cmake-3.25.3
+./configure
+make
+# Then to install it to your system finally do 
+sudo make install 
+
+# Do this to refresh your console 
+hash -r
+
+# Then finally check with
+cmake --version 
+```
+:::
+
+Install the development tools and the support libraries:
+
+```console
+sudo apt update && sudo apt install git git-lfs subversion build-essential cmake cmake-curses-gui cmake-qt-gui \
```

Comment:
Listing `cmake` here and building it above seems to conflict.

Considering that the version of CMake provided by `Ubuntu 23.04` is `3.25.1`[^1] and that the minimum required version by Slicer is `3.16.3`[^2], uninstalling the version provided by the operating system is probably not needed.

[^1]: https://packages.ubuntu.com/lunar/cmake
[^2]: https://github.com/Slicer/Slicer/blob/ba80bddd4a6cba9cfb82a527309cda30ada2ba6a/CMakeLists.txt#L1

Reply:
When I tried to use the installed cmake I got an error saying that some version were not accepted (including 3.25.1). I have not copied that error but I had to install a newer  release

Reply:
That is helpful details.

To move forward with this, could you add a section like the following ?

https://github.com/Slicer/Slicer/blob/ba80bddd4a6cba9cfb82a527309cda30ada2ba6a/Docs/developer_guide/build_instructions/linux.md#L48-L51

------

For context, it turns out that CMake `3.25.1` had a regression[^1] that was fixed in version `3.25.2` and above. This is why the default version can't be used.

[^1]: https://gitlab.kitware.com/cmake/cmake/-/issues/24567